dotcom-rendering/src/palette.ts

```diff
@@ -71,6 +71,14 @@ const headlineBackgroundColourDark = ({ design }: ArticleFormat): string => {
 	}
 };
 
+const headlineBorder = ({ design }: ArticleFormat): string => {
+	if (design === ArticleDesign.LiveBlog) {
+		return 'rgba(255,255,255, 0.2)';
+	}
+	if (design === ArticleDesign.DeadBlog) return '#CDCDCD';
+	return sourcePalette.neutral[86];
+};
+
```

Comment:
nit - would it be possible to make this a switch so it's consistent with the rest of the file?